.travis.yml

```diff
@@ -4,7 +4,7 @@ python:
   - "2.7"
   - "3.6"
 install:
-  - pip install bandit coveralls pytest pytest-cov pycodestyle
+  - pip install bandit coveralls pytest>=3.6 pytest-cov pycodestyle
```

Comment:
What does this do?

Reply:
I don't think it's relevant anymore; we thought it might solve the issue we're running into. Gonna undo.